hudi-common/src/test/java/org/apache/hudi/common/testutils/HoodieTestUtils.java

```diff
@@ -103,10 +103,17 @@ public static HoodieTableMetaClient init(Configuration hadoopConf, String basePa
   }
 
   public static HoodieTableMetaClient init(Configuration hadoopConf, String basePath, HoodieTableType tableType,
-                                           HoodieFileFormat baseFileFormat)
+                                           HoodieFileFormat baseFileFormat) throws IOException {
+    return init(hadoopConf, basePath, tableType, baseFileFormat, "org.apache.hudi.keygen.SimpleKeyGenerator", true);
```

Comment:
Isn't `SimpleKeyGenerator` the default keygen?

Reply:
yes, but I don't want to add a boolean and then add an if block to explicitly override the key gen value. If you have some compelling reason, let me know. I can do that.